src/main/scala/beam/utils/scenario/UrbanSimScenarioLoader.scala

```diff
@@ -57,6 +61,14 @@ class UrbanSimScenarioLoader(
   def loadScenario(): (Scenario, Boolean) = {
     clear()
 
+//    for {
+//      vehicleInfo <- vehicles
+//      vehicle = buildBeamVehicle(beamScenario.vehicleTypes, vehicleInfo, rand.nextInt)
+//    } {
+//      beamScenario.privateVehicles.put(vehicle.id, vehicle)
+//      vehicleInfo.initialSoc.foreach(beamScenario.privateVehicleInitialSoc.put(vehicle.id, _))
+//    }
+
```

Comment:
Comments like these are not very helpful in future, it is usually better to remove them